clientInit.js

```diff
@@ -0,0 +1,49 @@
+const __controllerTemplate = 'Controller';
+const __appVersion = __ver || 0;
+
+var controller = typeof(controller) === 'undefined' ? null : controller;
+var firstTimeLoad = typeof(firstTimeLoad) === 'undefined';
+
+// Establish a new session whether first time or because of expiry / server restart
+function __bindController (newController, sessionExpiration) {
+    if (controller && typeof(controller.shutdown) === 'function') {
+        controller.shutdown();
+    }
+    controller = newController;
+    console.log('Binding Controller');
```

Comment:
do we still want this console.log?

Reply:
I left it there for tracing any errors. As in, someone who was expecting to see 'creating bindster binding', will now see this message. I could remove it